lib/yao/resources/security_group.rb

```diff
@@ -3,6 +3,17 @@ module Yao::Resources
   class SecurityGroup < Base
     friendly_attributes :name, :description, :tenant_id
 
+    class << self
+      def all
+        self.list
+      end
```

Comment:
Thank you!
So, I noticed that those added methods are unnecessary. Because `RestfullyAccesible` provides  those methods features... :cry: